### DISCUSSION TITLE
Lotus daemon wont create .lotus directory

### DISCUSSION THREAD

**Discussion by PNetrox** [filecoin-project/lotus#11283] [Build and Sync]
OS: Ubuntu 20.04
Before all, i used:
`export LOTUS_PATH=~/.lotus`
I have issue with lotus daemon, when using first time:
`lotus daemon`
all i got is:
`2023/09/21 07:50:40 storess: Query
2023/09/21 07:50:40 storess: q.Prefix:
2023/09/21 07:50:40 storess: q.KeysOnly: true
2023/09/21 07:50:40 storess: q.Filters: 0
2023/09/21 07:50:40 storess: q.Orders: 0
2023/09/21 07:50:40 storess: q.Offset: 0`
And /.lotus directory still doesnt exist

**Comment by Reiers** [comment]
Hi @PNetrox 

By default it should be `.lotus` - so it's a hidden repo.
It should be there `cd .lotus`

Try setting `export LOTUS_PATH=/lotusdaemon` instead if you don't want the folder to be hidden. 

If you still encounter problems, please share more of the logs. Thanks!
